src/babble/babble.go

```diff
@@ -158,6 +160,11 @@ func (b *Babble) initNode() error {
 	return nil
 }
 
+func (b *Babble) initService() error {
+	b.Service = service.NewService(b.Config.ServiceAddr, b.Node, b.Config.Logger)
```

Comment:
We should make this initialization conditional, because as a User we may not want to start this service along Babble We should rely on the ServiceAddr length to know if we have to start it
IMHO I think we should leave this as a separated (debug) feature, and rely on the proxy to eventually get those information from Babble

Reply:
good point
cf new commit